src/components/Skeleton/index.tsx

```diff
@@ -61,30 +61,27 @@ const Span = styled.span`
 
 const randomInt = (min: number, max: number) => Math.floor(Math.random() * (max - min + 1)) + min;
 
+const formatValue = (value: string | number) => {
+  return typeof value === 'number' ? `${value}px` : value;
+};
+
+const StyledSpan = styled(Span)<SkeletonProps>`
+  ${({ count = 1, duration = 1.2, width = '100%', height = '100%', isCircle, isText, isPulsing = true }) => css`
```

Comment:
Do you need to add a default `count` here as well? There is already a default coming from `props` in line #81.

Reply:
It's not coming from there, I use it there just locally, as max value of loop. But ok, I can pass that value to `StyledSpan` and overwrite value from props.